personal-calls/frontend/src/services/Feature.ts

```diff
@@ -21,6 +23,8 @@ export default class FeatureService extends ObservableService<FeatureState> {
       EDGE_EXPERIMENT: Edge.DEFAULT,
       FEEDBACK_DIALOG: false,
       CREDIT_CAP: false,
+      DISABLE_WORKPASS: false,
+      ENABLE_FIN: false,
```

Comment:
We're not enabling this here is it?

Reply:
These are just default values. We enable flags with environment variables. All the flags should default to false, meaning "flag was not given in the environment"